var/spack/repos/builtin/packages/mysql/package.py

```diff
@@ -130,6 +131,11 @@ def cmake_args(self):
             options.append('-DBOOST_ROOT={0}'.format(spec['boost'].prefix))
         if '+client_only' in self.spec:
             options.append('-DWITHOUT_SERVER:BOOL=ON')
+        options.append('-DWITH_EDITLINE=system')
```

Comment:
Shouldn't this be removed then?

Reply:
If you do not set it to `system`, then it doesn't work, because it takes the `bundled` option which uses the `editline` that comes with mysql.
You need to set to `system` and set the `editline` path, so it replaces the system one.

Reply:
Ah, that makes sense, sorry. I just ran another build on Linux and everything seems to work so far.